feat: validate values for `cache-control` and `content-type` headers in dev mode (#13114)
sveltejs/kit: .changeset/rich-pants-beam.md

---
'@sveltejs/kit': minor
---

> feat: add header validation warnings in dev mode
> 
> Adds validation for common headers (cache-control, content-type)
> in dev mode to help catch invalid values early.

Reviewer: ```suggestion
feat: validate values for `cache-control` and `content-type` headers in dev mode
```
